go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containers/image/v5
 
-go 1.20
+go 1.21
```

Comment:
My understanding is that along with this, we also need toolchain 1.21 as well? Please double-check.

Reply:
My reading of “Go toolchain selection” in https://go.dev/doc/toolchain is that it is not compulsory, and effectively defaults to the `go` directive.

I don’t mind adding `1.21.0` here, if you think it would be more explicit. (Or 1.21.$latest ? That would force users to update.)